src/test/java/edu/ucmo/spring_example/controller/CarControllerTest.java

```diff
@@ -32,6 +32,12 @@
     @Autowired
     private JdbcTemplate template;
 
+    @Test
+    public void findOneThatDoesNotExist() {
+        Car car = carController.getOne(99999);
+        assertNull(car);
+    }
```

Comment:
Is this test sufficient?

Reply:
I don't know, doi you have a suggestion for another one?

Reply:
No, it's OK.